pkg/storaged/btrfs/btrfs-tool.py

```diff
@@ -95,59 +63,47 @@ def list_filesystems():
 tmp_mountpoints = set()
 
 
-def add_tmp_mountpoint(db, fs, dev, opt_repair):
+def get_tmp_mountpoint_path(uuid):
+    return os.path.join(TMP_MP_DIR, str(os.getpid()) + ":" + uuid)
```

Comment:
OOI, why? If two different processes mount the fs onto the same dir, that doesn't matter? They won't be visible to each other and to the outside anyway.

Reply:
Yes, this is just for removing the directory itself during cleanup. I think I'll push a followup commit that doesn't remove the directories, that should simplify this further.